.gitlab-ci.yml

```diff
@@ -13,6 +13,8 @@ include:
     file: '.gitlab-ci-check-commits.yml'
   - project: 'Northern.tech/Mender/mendertesting'
     file: '.gitlab-ci-check-license.yml'
+  - project: 'Northern.tech/Mender/mendertesting'
+    file: '.gitlab-ci-check-docker-build.yml'
```

Comment:
Why including the template if we are overriding `build:docker` ?

Reply:
argh sorry, that's a leftover - my intent was to use the standardized approach, however the `build:docker` job seems to not be used anywhere in the pipeline, I forgot to remove the template (preferably I would also remove the `build:docker` job, but I'm not sure if you might want to keep it to just test if that `make` target still works)

Reply:
While I see your good intentions, I think for this case is better to use the `Makefile` from the CI to guarantee we don't break it :+1: